lib/src/config/config.rs

```diff
@@ -574,7 +574,8 @@ impl ClientConfig {
     /// Instantiates the Nimiq client from this configuration
     ///
     pub async fn instantiate_client(self) -> Result<Client, Error> {
-        Client::from_config(self).await
+        let client: Result<Client, Error> = Client::from_config(self).await;
+        client
```

Comment:
This change seems to be unnecessary on my environment, it was failing for you if you didn't specify the type for `client` explicitly, right?

Reply:
I thought so but now that I changed it, it worked